expyfun/_experiment_controller.py

```diff
@@ -843,7 +852,8 @@ def flip(self, when=None):
         # this waits until everything is called, including last draw
         gl.glClear(gl.GL_COLOR_BUFFER_BIT)
         gl.glBegin(gl.GL_POINTS)
-        gl.glColor4f(0, 0, 0, 0)
+        if not self._enable_video:
+            gl.glColor4f(0, 0, 0, 0)
```

Comment:
That would have a nice side benefit of making it clear that you shouldn't combine draws and video display